lib/YaST/Module.pm

```diff
@@ -42,7 +42,7 @@ sub open {
         );
     }
     elsif ($ui eq 'qt') {
-        launch_yast2_module_x11($module, extra_vars => get_var('YUI_PARAMS'));
+        launch_yast2_module_x11($module, extra_vars => get_var('YUI_PARAMS'), maximize_window => 1);
```

Comment:
you need to pass this as parameter (and document it on the top) existing code might get broken needles if we maximize as default.